sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/context/AuthTokenContext.java

```diff
@@ -125,6 +125,7 @@ public void clearToken() {
         for (Listener listener : mListeners) {
             listener.onNewAuthToken(null);
             listener.onNewUser(null);
+            listener.onSignOut();
```

Comment:
It will be raised on disable Identity module

Reply:
Why do you need the separate method then? Maybe we can just re-use the previous one with `null` in parameters?

Reply:
Disables already uses null indeed.